spec/stats/kbuild/02

```diff
@@ -0,0 +1,378 @@
+2024-02-26 06:47:23 cd /lkp/benchmarks/kbuild/kbuild/1
+2024-02-26 06:47:23 make mrproper
+2024-02-26 06:47:27 make defconfig
+  HOSTCC  scripts/basic/fixdep
+  HOSTCC  scripts/kconfig/conf.o
+  HOSTCC  scripts/kconfig/confdata.o
+  HOSTCC  scripts/kconfig/expr.o
+  LEX     scripts/kconfig/lexer.lex.c
+  YACC    scripts/kconfig/parser.tab.[ch]
+  HOSTCC  scripts/kconfig/lexer.lex.o
+  HOSTCC  scripts/kconfig/parser.tab.o
+  HOSTCC  scripts/kconfig/preprocess.o
+  HOSTCC  scripts/kconfig/symbol.o
+  HOSTLD  scripts/kconfig/conf
```

Comment:
for above lines (from line 4 to line 14), pls only leave one line like "HOSTCC  scripts/basic/fixdep" which is enough for the spec.

Reply:
ok